### ISSUE TITLE
Taco Conf LA

### ISSUE THREAD

**Issue by jcblw** [TacoConf/2015#3]
Super excited for this.

I know some bomb taco places in Santa Monica/ Venice
- Tacos Por Favor
- The Flying Jalapeño
- Holy Guacamole ( this place is good but small )

I know some venues like 
- http://fullcirclevenice.com/

@motdotla do you have contacts there 

cc: @dguttman @emkay @thedaniel

**Comment by zenorocha** [comment]
Yay!

Never been to a TacoConf, I'm curious about it ;)

**Comment by emkay** [comment]
Tacos Punta Cabras on SM Blvd in Santa Monica. :+1: 

Also maybe @jxson can give us that path on the east side.

**Comment by emkay** [comment]
@jcblw if we do it on the West side maybe we could hit up some taco places and make our way to the beach. do some talks there. no projectors, just talks.

**Comment by jcblw** [comment]
@emkay that's a sick idea, like a drum circle in Venice

**Comment by jxson** [comment]
An old route @nlacasse thought up from a while ago: https://www.google.com/maps/d/viewer?mid=z9Lms0Tru7W8.k-wmFEeAaIEQ

**Comment by soldair** [comment]
can't skip olvera street. Its the birthplace of la tacos.

**Comment by mikeal** [comment]
Is there a place you can stop to do talks in? We did talks the first year and they were amazing, and a welcome break from the constant bike ride :)

Most of the talks were slideless so you could even do them in a park or something like that.

**Comment by emkay** [comment]
As much as doing talks on the beach would be cool, I think we really need to go east for the tacos. We can map out some parks, etc. for talks.

**Comment by jcblw** [comment]
Its summer time in LA how do you not go the beach. Im down for either, Im afraid that if its east side Im probably not going as big of a help.

**Comment by emkay** [comment]
@jcblw yeah :( I kinda want west side too because of beach. Just thinking of the tacos. Maybe not as big of a deal.

**Comment by emkay** [comment]
ok the verdict is in. for this round we are going to focus on the "east" since that is where tacos reign supreme and it is Taco Conf after all.

**Comment by soldair** [comment]
tacos do not reign supreme on the east side guarantee it. #heartbroken

**Comment by soldair** [comment]
it'll be great though I'm sure. looking forward to it. if you want real mexican food you can metro it to union station after

**Comment by emkay** [comment]
I mean kinda follow the path that @jxson linked. We can for sure head downtown.
